extras/nixpkgs.nix

```diff
@@ -0,0 +1,8 @@
+{inputs, ...}: {
+  options.perSystem = mkPerSystemOption ({ pkgs, system, ... }: {
+    imports = [
+      "${inputs.nixpkgs}/nixos/modules/misc/nixpkgs.nix"
```

Comment:
`inputs.nixpkgs` makes an assumption to the name of user's inputs, which is bad. We still need to solve #157 to properly use an input.